### PR TITLE
Create devcontainer.json

### PR DESCRIPTION
**Pull request by Jordanwaslistening**
"ghcr.io/flexwie/devcontainer-features/op:1": {}

### CODE REVIEW

File: .devcontainer/devcontainer.json
Comment (Jordanwaslistening) [inline .devcontainer/devcontainer.json:1]: 🤫

**Review by Jordanwaslistening — COMMENTED** [review]
Looks good.

**Review by Jordanwaslistening — COMMENTED** [review]
🤫

**Comment by bamurtaugh** [comment]
Thanks for opening! As mentioned in another PR, perhaps this PR was intended for another repo, since this repo is for spec/documentation content? If you have other questions/feedback/ideas, please let us know.

**Pull request closed without merging**